packages/fxa-payments-server/src/components/PaymentErrorView/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Localized } from '@fluent/react';
+import { StripeError } from '@stripe/stripe-js';
+
+import { getErrorMessage } from '../../lib/errors';
+import errorIcon from './error.svg';
+import SubscriptionTitle from '../SubscriptionTitle';
+
+import './index.scss';
+
+type GeneralError = {
+  code: string;
+};
+
+export type PaymentErrorViewProps = {
+  error: undefined | StripeError | GeneralError;
+  onRetry: Function;
+  className?: string;
+};
+
+export const PaymentErrorView = ({
+  error,
+  onRetry,
+  className = '',
+}: PaymentErrorViewProps) => {
+  return error ? (
+    <>
+      <SubscriptionTitle screenType="error" className={className} />
+      <section
+        className={`container card payment-error ${className}`}
+        data-testid="payment-error"
+      >
+        <div className="wrapper">
+          <img id="error-icon" src={errorIcon} alt="error icon" />
+          <div>
+            <Localized id={getErrorMessage(error.code || 'UNKNOWN')}>
+              <p data-testid="error-payment-submission">
+                {getErrorMessage(error.code || 'UNKNOWN')}
```

Comment:
Wouldn't this display an id, not an error message?  I don't think that's all that meaningful to the users.  Maybe we should just skip the fallback content here.

Reply:
"UNKNOWN" here returns "BASIC_ERROR" message.

```
basic-error-message = Something went wrong. Please try again later.
```

Reply:
I don't disagree @chenba, but this isn't something @meandavejustice introduced (see: https://github.com/mozilla/fxa/pull/7718/files#diff-ebb0774ad95dbf124b161dacb435e38f76977a9770ed80228d063ecc3bf85b2dL267). We could add a clean-up issue?

Reply:
Sure I can file an issue.